### PR TITLE
Publish the most recent model downloader revision

### PR DESCRIPTION
**Pull request by IRDonch**

### CODE REVIEW

**Comment by IRDonch** [comment]
CC @atsyplen

File: model_downloader/README.md [hunk 1]
```diff
@@ -78,6 +84,21 @@ Usage
...
+   se-resnext-101
+   Sphereface
+   license-plate-recognition-barrier-0007
+   mobilenet-v1-1.0-224
+   mobilenet-v2
+   faster_rcnn_inception_v2_coco
+   deeplabv3
+   ctpn
+   ssd_mobilenet_v1_coco
+   faster_rcnn_resnet101_coco
```
Comment (kchechil) [inline model_downloader/README.md]: do not see mobilenet-v2-1.4-224, did I miss something?

File: model_downloader/downloader.py [hunk 2]
```diff
@@ -93,38 +228,31 @@ def get_confirm_token(response):
...
-    destination = os.path.join(path, name)
-    chunk_size = 32768
-    with open(destination, 'wb') as f:
-        process_download(chunk_size, response, size, f)
-    print(name, '====>', destination)
-
 def delete_param(model):
-    tmpfile = os.path.join(args.output_dir, 'tmp.txt')
-    with open(model, 'r') as input_file, open(tmpfile, 'w') as output_file:
+    tmpfile = args.output_dir / 'tmp.txt'
```
Comment (kchechil) [inline model_downloader/downloader.py]: will it work under Windows?

File: model_downloader/downloader.py [hunk 3]
```diff
@@ -93,38 +228,31 @@ def get_confirm_token(response):
...
         data=input_file.read()
         updated_data = re.sub(' +save_output_param \{.*\n.*\n +\}\n', '', data, count=1)
         output_file.write(updated_data)
-    shutil.move(tmpfile, model)
+    tmpfile.replace(model)
 
 def layers_to_layer(model):
-    tmpfile = os.path.join(args.output_dir, 'tmp.txt')
-    with open(model, 'r') as input_file, open(tmpfile, 'w') as output_file:
+    tmpfile = args.output_dir / 'tmp.txt'
```
Comment (kchechil) [inline model_downloader/downloader.py]: will it work under Windows?

File: model_downloader/downloader.py [hunk 4]
```diff
@@ -93,38 +228,31 @@ def get_confirm_token(response):
...
         output_file.write(updated_data)
-    shutil.move(tmpfile, model)
+    tmpfile.replace(model)
 
 def change_dim(model, old_dim, new_dim):
     new = 'dim: ' + str(new_dim)
     old = 'dim: ' + str(old_dim)
-    tmpfile = os.path.join(args.output_dir, 'tmp.txt')
-    with open(model, 'r') as input_file, open(tmpfile, 'w') as output_file:
+    tmpfile = args.output_dir / 'tmp.txt'
```
Comment (kchechil) [inline model_downloader/downloader.py]: Windows?

File: model_downloader/downloader.py (hunk 2 above)
Comment (IRDonch) [reply model_downloader/downloader.py]: Yes, `pathlib` correctly handles path concatenation on both Windows & Linux.

**Comment by IRDonch** [comment]
Jenkins please retry a build

**Comment by atsyplen** [comment]
Jenkins please retry a build

**Pull request merged**